.github/workflows/build.yml

```diff
@@ -218,7 +218,7 @@ jobs:
         if: failure()
         uses: actions/upload-artifact@v4
         with:
-          name: failed-admin-tests-screenshots
+          name: failed-admin_${{ matrix.ci_node_index }}-tests-screenshots
```

Comment:
Using underscore here to represent different node with respect to spec type in a single word way like admin_0, admin_1 etc